index.js

```diff
@@ -116,6 +117,7 @@ function do_bundle(file, opts){
                 if( opts.arithmetic ) {
                     jspm_input += ' ' + opts.arithmetic.trim();
                 }
+                jspm_input = jspm_input.replace(/\\/g, '/');
```

Comment:
This will mistakenly replace `\` with a `/` for paths that includes a `\` on unix systems.
I'm not familiar with dealing with this issue but shouldn't it be handled by jspm?